sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2079,7 +2079,7 @@ object SQLConf {
     .internal()
     .doc("When true, outer CTE definitions takes precedence over inner definitions.")
```

Comment:
Now, it became `three-state conf`. Shall we mentioned more about `false` and `empty`?

Reply:
Sounds we should.

Reply:
Sure, done in 7249404.